vendor/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -146,13 +181,35 @@ def parse(input_path, output_path, layout_dir, clean=False):
         files = [os.path.join(input_path, file_name) for file_name
                  in file_names]
 
+    # Skipped plist reports from html generation because it is not a
+    # plist file or there are no reports in it.
+    skipped_report = set()
+
+    # Source files which modification time changed since the last analysis.
+    changed_source_files = set()
+
     html_builder = HtmlBuilder(layout_dir)
     for file_path in files:
-        plist_to_html(file_path, output_path, html_builder)
+        print(file_path)
+        sr, changed_source = plist_to_html(file_path,
+                                           output_path,
+                                           html_builder)
+        if changed_source:
+            changed_source_files = changed_source_files.union(changed_source)
+        if sr:
+            skipped_report.add(sr)
 
     print('\nTo view the results in a browser run:\n> firefox {0}'.format(
         output_path))
 
+    if changed_source_files:
+        changed_files = '\n'.join([' - ' + f for f in changed_source_files])
+        print("\nThe following source file contents changed since the "
```

Comment:
Statics are available at `parse` command. Please remove the following sentence from this log message: `Multiple reports were not shown and skipped from the statistics.`.